lms/djangoapps/courseware/tests/test_courses.py

```diff
@@ -432,7 +432,8 @@ def test_completion_ignores_non_scored_items(self):
         Test that we treat a sequential with incomplete (but not scored) items (like a video maybe) as complete.
         """
         course = CourseFactory()
-        chapter = ItemFactory(parent=course, category='chapter', graded=True, due=datetime.datetime.now())
+        chapter = ItemFactory(parent=course, category='chapter', graded=True, due=datetime.datetime.now(),
```

Comment:
This was creating a default start date of 2030-01-01, which made it unreleased. So I made it released an hour ago. Is that unreasonable? It means that any unreleased material is not counted as complete, even if it is not scored.

Reply:
Ah yeah, we do that to avoid releasing content accidentally. I think that's a reasonable workaround. You could probably also just put in a hardcoded date like 2000-01-01 or something.